pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -51,6 +55,23 @@ message ClientKeepaliveConfiguration {
   bool permit_without_stream = 3;
 }
 
+message ClientOAuthConfiguration {
+  oneof credentials {
+    // Use default google credentials (false by default)
```

Comment:
The 'false by default' part can be removed now, because that's enforced by the use of `oneof`.
Nit: Capitalize 'google'.

Reply:
done